xain/fl/participant/participant_test.py

```diff
@@ -1,10 +1,10 @@
 import numpy as np
 import pytest
 
-from xain.benchmark.net import model_fns
+# from benchmarks.benchmark.net import model_fns  # FIXME refactor
```

Comment:
Should this be fixed now or is there any task created to solve this?

Reply:
We're in the process of refactoring how model loading works substantially, so I'd say we don't need to invest the effort to fix this now only to re-work it next week. I've created a ticket to not loose track of it: XP-184